### DISCUSSION TITLE
ul ol indentation problem

### DISCUSSION THREAD

**Discussion by blueMountain007** [payloadcms/payload#982] [Q&A]
I wanted to control the indentation by tab, but I failed. Later found left-indent and right-indent, there are 2 problems:
1. After right-indent, the symbol on the left does not change. It would be perfect if it could be changed.
2. After left-indent, the original small dot on the left disappears.

<img width="727" alt="image" src="https://user-images.githubusercontent.com/16400882/185825161-eb8c87d7-bde4-49bc-9425-8cbd48296e57.png">

when i click left indent:

<img width="675" alt="image" src="https://user-images.githubusercontent.com/16400882/185825236-9f4129bf-7f54-4cc0-97fd-ab6ce86926b2.png">

You can test it in this [demo site](https://demo.payloadcms.com/admin/collections/posts/6302d51365a37149b6f30864).

1. Can indent-left keep the small dot symbol on the left still there?
2. Whether the indentation can be controlled by tabs in the payload, like contentful currently controls indentation by tabs
<img width="802" alt="image" src="https://user-images.githubusercontent.com/16400882/185825619-53acb4ca-e013-48b8-96e3-d78272daa40d.png">

**Comment by alfienguyen** [comment]
For reference, this library built on top of Slate does have the mentioned functionalities: https://plate.udecode.io/docs/plugins/list

**Comment by jamov** [comment]
There's an open issue about lists & indents on the editor https://github.com/payloadcms/payload/issues/551
